### ISSUE TITLE
Bug in documentation: Dialog (Modal)/keyboard interaction/Note/tabindex

### ISSUE THREAD

**Issue by iamrafan** [w3c/aria-practices#925]
https://www.w3.org/TR/wai-aria-practices/#issue-container-generatedID-7

The third bullet in 1st point says:

>If content is large enough that focusing the first interactive element could cause the beginning of content to scroll out of view, it is advisable to add ```tabindex=-1``` to a static element at the top of the dialog, such as the dialog title or first paragraph, and initially focus that element.

Expected:

```tabindex=0``` should be used to make the static content focusable.

**Comment by mcking65** [comment]
@iamrafan, no we intentionally use `tabindex="-1"` so the static element is focusable but not part of the tab sequence. This allows the JavaScript to focus the static element when the dialog opens, which also causes screen reader focus to start there, but prevents a tab stop on a static element.

**Comment by iamrafan** [comment]
@mcking65 thanks for the clarification, are you going to change the wording to make it more clear?

I have one other question about the inconsistency in guidance about where the focus should go when the modal dialog is closed:

The [keyboard interaction for modal dialog](https://www.w3.org/TR/wai-aria-practices/#issue-container-generatedID-7) says: when a dialog is closed, the focus can be set on another element that provides logical work flow

![image](https://user-images.githubusercontent.com/22068434/46833578-cd035380-cd5d-11e8-877a-c4a7260b45b9.png)

But 

The [common failure technique F85](https://www.w3.org/WAI/WCAG21/Techniques/failures/F85) says it is a WCAG failure if the focus does not go back to the trigger control or the control next to it when the dialog is closed

![image](https://user-images.githubusercontent.com/22068434/46833445-89104e80-cd5d-11e8-82c7-cae5e4cdc1d0.png)

Which of them is correct?

**Comment by jnurthen** [comment]
Firstly please log new issues for things like this. This issue now has confusing extra information in it.

For this issue I suggest logging an issue against WCAG F85 
A good example where this would not apply would be a cancel button which opens a confirmation dialog. This cancel button would likely take to a screen where that cancel button is no longer present.

**Comment by iamrafan** [comment]
@jnurthen Thanks, sorry for the confusion.

**Comment by carmacleod** [comment]
@iamrafan Regarding your first question:
> thanks for the clarification, are you going to change the wording to make it more clear?

The wording is already pretty clear. Setting tabindex="-1" is defined in the [HTML spec for tabindex](https://www.w3.org/TR/html52/editing.html#the-tabindex-attribute) as follows:

> For non-interactive elements that need to receive focus but that are not meant to be part of the sequential focus navigation order (for instance, the target of a skip link, or a container element that needs to be programmatically focused via JavaScript), authors should use a negative value of tabindex="-1".

**Comment by iamrafan** [comment]
@iamrafan Sure. I was just wondering if I am good to close this issue or is it being worked on, because its tagged as feedback and to "1.1 APG Release 3" milestone

**Comment by iamrafan** [comment]
> Firstly please log new issues for things like this. This issue now has confusing extra information in it.
> 
> For this issue I suggest logging an issue against WCAG F85
> A good example where this would not apply would be a cancel button which opens a confirmation dialog. This cancel button would likely take to a screen where that cancel button is no longer present.

**Comment by carmacleod** [comment]
@iamrafan Good point - perhaps @mcking65 was planning to add some words about screen reader focus.

@mcking65 If you like the following wording (added some words to end of sentence), I can provide a PR:
> If content is large enough that focusing the first interactive element could cause the beginning of content to scroll out of view, it is advisable to add tabindex=-1 to a static element at the top of the dialog, such as the dialog title or first paragraph, and initially focus that element programmatically to cause AT focus to start there.
